README.md

```diff
@@ -9,23 +9,23 @@
 
 <!--END STABILITY BANNER-->
 
-This module exports a single class called `KubectlV24Layer` which is a `lambda.LayerVersion` that
+This module exports a single class called `KubectlV25Layer` which is a `lambda.LayerVersion` that
 bundles the [`kubectl`](https://kubernetes.io/docs/reference/kubectl/kubectl/) and the
 [`helm`](https://helm.sh/) command line.
 
-> - Helm Version: 3.9.4
-> - Kubectl Version: 1.24.0
-> 
+> - Helm Version: 3.10.3
```

Comment:
Shoot, i think we want Helm version 3.11.1, that supports kubectl v1.25 too, right?

Reply:
Checking [here](https://helm.sh/docs/topics/version_skew/#supported-version-skew) again, looks like I missed Helm 3.11.x. Oops. Will fix that in a bit.

Reply:
I've updated helm support to 3.11.1. Please find all tests attached:

![deploy-01](https://user-images.githubusercontent.com/11181378/222526053-262b6bee-8bb8-4732-8b9e-7d2d4230ea5d.png)

![deploy-04](https://user-images.githubusercontent.com/11181378/222526118-82b2da68-70f2-45cd-a29b-7930dd39dd2f.png)